packages/renderer/src/lib/dialogs/Dialog.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
```

Comment:
Do we want to expose it in the ui library ?

Reply:
I think we should, I just didn't want to start there since I was aware of any imminent extension need. i.e. get it in, use in a few more places and make sure it's ok, then export.